models/intel/text-spotting-0002/description/text-spotting-0002.md

```diff
@@ -48,5 +49,53 @@ This model is a Mask-RCNN-based text detector with ResNet50 backbone and additio
     classes for every output bounding box.
 5.  Name: `text_features`, shape [100x64x28x28]. Text features that are fed to a text recognition head.
 
+
+## Encoder model specification
+
+The text-spotting-0002-recognizer-encoder model is a fully-convolutional encoder of text recognition head.
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| Word spotting hmean ICDAR2015, without a dictionary | 59.04%    |
```

Comment:
on my view, this metric does not correspond to model part, it is accuracy of whole pipeline, so should be moved to level up

Reply:
possibly framework and type too relates to whole model

Reply:
Added part composite model specification